### ISSUE TITLE
Wrong exception type in cvxprog.py

### ISSUE THREAD

**Issue by paperstiger** [cvxopt/cvxopt#137]
On line 515 and 1741 or file cvxprog.py, the exception should be ValueError instead of ValueEror

**Comment by martinandersen** [comment]
Do you mean *TypeError* instead of ValueError on line 515? There is no exception on line 1741.

**Comment by paperstiger** [comment]
Yes. It is on master branch. But probably it does not matter since you are raising exception anyway.
